src/main/java/core/basesyntax/FieldXY.java

```diff
@@ -13,7 +15,106 @@
 public class FieldXY {
 
     public void moveRobot(Robot robot, int toX, int toY) {
-        //your code
+        int x = toX - robot.getX();
+        int y = toY - robot.getY();
+        if (x != 0) {
```

Comment:
What's this check for?

Reply:
x is the difference between current robot coordinate and target coordinate. this check if we need to move by x ordinate.

Reply:
if x is 0 your robot won't move anyway, as you have a `for` loop at move methods with `i < x` check. So this check is redundant and tests will pass without them anyway